src/node_main_instance.cc

```diff
@@ -99,6 +99,8 @@ ExitCode NodeMainInstance::Run() {
   CHECK_NOT_NULL(env);
 
   Context::Scope context_scope(env->context());
+  env.get()->InitializeEbpf();
+
```

Comment:
Though this seems like a good location to do this, due how N|Solid is implemented, we should move it a bit earlier: to EnvList initialization.